test/fixtures/s2-geometry.js

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2020 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+export const data =
+  's2,value\n' +
```

Comment:
should add a row contain invalid s2 token

Reply:
null values are added here: https://github.com/keplergl/kepler.gl/pull/800/files/980b1d9b65a619a698281d25535ceafa98b48c20#diff-5dc6b0fd7be83b659ba153670b303cf9R88

before performing the tests.
I am gonna add a invalid s2 id and update the test

Reply:
after talking to @Pessimistress, there is not really a way to test whether an s2 token is valid or not.

Reply:
I added some invalid s2 tokens to make sure kepler will not crash